src/PerzApiPhpClient.php

```diff
@@ -118,19 +119,21 @@ public function getEntities(array $data) {
         'origin' => $data['origin'] ?? NULL,
         'language' => $data['language'] ?? NULL,
         'view_mode' => $data['view_mode'] ?? NULL,
+        'content_uuid' => $data['content_uuid'] ?? NULL,
         'q' => $data['q'] ?? NULL,
         'content_type' => $data['content_type'] ?? NULL,
         'tags' => $data['tags'] ?? NULL,
         'all_tags' => $data['all_tags'] ?? NULL,
         'date_start' => $data['date_start'] ?? NULL,
         'date_end' => $data['date_end'] ?? NULL,
-        'rows' => $data['rows'] ?? 10,
-        'start' => $data['start'] ?? 0,
-        'sort' => $data['sort'] ?? 'modified',
-        'sort_order' => $data['sort_order'] ?? 'desc',
+        'rows' => $data['rows'] ?? NULL,
+        'start' => $data['start'] ?? NULL,
+        'sort' => $data['sort'] ?? NULL,
+        'sort_order' => $data['sort_order'] ?? NULL,
       ],
     ];
-    return $this->request('get', $uri, $options);
+    $client = ObjectFactory::getGuzzleClient(['base_uri' => $this->baseUrl]);
+    return $client->get($uri, $options);
```

Comment:
Why is it created another client inside of a client? Is it to reset the `base_uri` or remove the `HmacAuthMiddleware` handler? If so, do all of the other diffs need to happen?

Reply:
Initially when this client was developed, GET api needed authentication but later it was made public. Hence now we don't receive X-Server-Authorization-Hmac-Sha256 header in response. If we use the current client it will throw an error  The recent change is to use the guzzle client without the middleware.

Reply:
Gotcha! Thanks for the information.